src/components/data-management/DownloadData.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect } from 'react';
+import _ from 'lodash';
+import {
+  Menu, Tooltip, Dropdown, Button,
+} from 'antd';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { saveAs } from 'file-saver';
+import pipelineStatus from '../../utils/pipelineStatusValues';
+import downloadData from '../../utils/data-management/downloadExperimentData';
+import downloadTypes from '../../utils/data-management/downloadTypes';
+import { exportQCParameters, filterQCParameters } from '../../utils/exportQCParameters';
```

Comment:
Functions in `exportQCParameters` are only used in this file, so I suggest removing them from utils and moving them in here

Reply:
I would move this `exportQCParameters` file to the data management folder in utils